html/admin/api/featured/articleSearch.php

```diff
@@ -4,7 +4,7 @@
 if (!isset($_POST['term'])) finish(false, ["code" => "PARAM", "message"=> "No term set"]);
 $term = $bCMS->sanitizeString($_POST['term']);
 if (strlen($term) > 0) $DBLIB->where("(articlesDrafts.articlesDrafts_headline LIKE '%" . $bCMS->sanitizeString($term) . "%')");
-if (isset($_POST['categoryid'])) $DBLIB->where("FIND_IN_SET('" . $bCMS->sanitizeString($_POST['categoryid']) . "',articles_categories)");
+if (isset($_POST['categoryid'])) $DBLIB->where("articles.articles_id IN (SELECT articles_id FROM articlesCategories WHERE categories_id = " . $_POST['categoryid'] . ")");
```

Comment:
Instead of concatenating strings I think we should use the `?` syntax where possible. E.g. https://github.com/yorknouse/website/blob/astro-mobile-navbar/html/frontend/api/searchSuggestions.php#L47

This seems fine for now but worth making a ticket for the future to replace these

Reply:
Ohh I see what you mean. Yeah the style on the backend is inconsistent. If you could add a ticket about that to the board that would be great